static/js/activity.js

```diff
@@ -210,6 +210,12 @@ export function send_presence_to_server(want_redraw) {
                 presence.set_info(data.presences, data.server_timestamp);
                 redraw();
             }
+
+            if (resend_request) {
+                setTimeout(() => {
+                    send_presence_to_server(want_redraw, true);
+                }, ACTIVE_PING_INTERVAL_MS);
+            }
```

Comment:
We need to do this in the `error` callback too, or the moment you get a transient error, this will stop being called at all.

Probably also a good idea to do this step first, so that errors inside the success/error handler can't prevent this from being scheduled.